frontend/src/boot/axios.js

```diff
@@ -7,6 +7,20 @@ const axiosInstance = axios.create({
   baseURL: process.env.BASE_URL
 });
 const _appendSubdomain = (domain, subdomain) => {
```

Comment:
When possible, try not to overload functions with a lot of different steps. Ideally, a function would do only one thing and the name of the function would describe the one thing it does.